src/components/vacation-requests-table/vacation-requests-table-toolbar/toolbar-update-status-button.tsx

```diff
@@ -16,25 +16,23 @@ interface UpdateStatusButtonProps {
   ) => Promise<void>;
 }
 
-/**
- * Status update button component
- *
- * @param props component properties
- */
 const UpdateStatusButton = ({
   buttonType,
   selectedRowIds,
   updateVacationRequestStatus
 }: UpdateStatusButtonProps) => {
-  /**
-   * Handle update vacation status
-   */
+  const isApprove = buttonType === VacationRequestStatuses.APPROVED;
   const handleUpdateVacationRequestStatus = async () => {
     await updateVacationRequestStatus(buttonType, selectedRowIds);
-  };
-
+  }; 
   return (
-    <Button variant="contained" fullWidth onClick={handleUpdateVacationRequestStatus}>
+    <Button 
+    variant="contained"
+    fullWidth
+    onClick={handleUpdateVacationRequestStatus}
+    sx={{backgroundColor: isApprove ? "#4caf50" : "#f44336",
+        "&:hover": {
+          backgroundColor: isApprove ? "#45a049" : "#da190b"}}}>
```

Comment:
Fix indentation, this means your linter is not working properly, it should be set to format on save so this happens automatically